src/main/java/cloud/eppo/EppoHttpClient.java

```diff
@@ -41,34 +41,22 @@ private static OkHttpClient buildOkHttpClient() {
     return builder.build();
   }
 
-  // TODO: use this for Java, callback for Android; clean as needed
-  public Response get(String path) {
-    HttpUrl httpUrl =
-        HttpUrl.parse(baseUrl + path)
-            .newBuilder()
-            .addQueryParameter("apiKey", apiKey)
-            .addQueryParameter("sdkName", sdkName)
-            .addQueryParameter("sdkVersion", sdkVersion)
-            .build();
+  public byte[] get(String path) {
+    Request request = buildRequest(path);
 
-    Request request = new Request.Builder().url(httpUrl).build();
-    try {
-      return client.newCall(request).execute();
+    try (Response response = client.newCall(request).execute()) {
```

Comment:
_try-with-resources_ block. Modern java is neat.

Reply:
I love how something added in 2011 is "modern" 😛